docs/website/docs/getting-started/pytorch.md

```diff
@@ -0,0 +1,100 @@
+# PyTorch Integration
+
+IREE supports compiling and running PyTorch programs represented as
+`nn.Module` [classes](https://pytorch.org/docs/stable/generated/torch.nn.Module.html) as well as models defined using [`functorch`](https://pytorch.org/functorch/).
+
+## Prerequisites
+
+Install IREE pip packages, either from pip or by
+[building from source](../building-from-source/python-bindings-and-importers.md):
+
+```shell
+pip install \
+  iree-compiler \
+  iree-runtime \
+  iree-tools-tf
```

Comment:
Is iree-tools-tf needed?

Reply:
Whoops, a small bit of leftover copy pasting!  Removed.